inc/wp-forms-api.js

```diff
@@ -13,13 +13,49 @@
 		},
 	});
 
+	function loadMCETemplates(content, editor) {
+		wp.ajax.post('wp_form_get_mce_templates', {
+			content: content,
+			type: getUserSetting('editor')
+		}).done(function(result) {
+			_.each(result, function(html, id) {
+				var $box = $('#' + id);
+
+				$box.html(html);
+
+				if(editor) {
+					$box.find('.wp-editor-wrap').addClass(editor == 'html' ? 'html-active' : 'tmce-active');
+				}
+
+				var config = _.clone(tinyMCEPreInit.mceInit.content);
+
+				config.selector = id;
+				config.toolbar1 = ['bold', 'italic', 'underline', 'blockquote', 'strikethrough', 'bullist', 'numlist', 'alignleft', 'aligncenter', 'alignright', 'undo', 'redo', 'link', 'unlink'].join(',');
```

Comment:
Shouldn't these come from WP somehow? Since normally the toolbar can be filtered I believe...